### ISSUE TITLE
Update to use latest Paket?

### ISSUE THREAD

**Issue by dsyme** [fsprojects/IfSharp#130]
The latest Paket allows the paket.dependencies to contain 

    generate_load_scripts: true

See https://fsprojects.github.io/Paket/dependencies-file.html#Generate-load-scripts

This is simpler. Let's update to use this

**Comment by cgravill** [comment]
Looks good and will mean consistency between script and terminal usage. Saw the plans to implement this and great it's merged. From the releases it looks like it's only in the beta/pre-release:

https://github.com/fsprojects/Paket/releases/tag/4.0.0-beta006

For a project like IfSharp I suggesting holding until the feature is in general release. I'd also really like to see something like @isaacabraham suggested on fsprojects/Paket#1943 to have an easier to remember top level include script. I put in Paket.Generated.Refs.fsx for this purpose but would much rather use something directly from Paket.

**Comment by smoothdeveloper** [comment]
@dsyme, can someone explains to me the way it works in terms of folder structure?

I confirm it is only in prerelease, and we'd like to update the folder structure for a shallower one (and potentially one which is platform agnostic when run by default), it would be nice to resume discussion there https://github.com/fsprojects/Paket/issues/1943 as I had some reserves about where to put stuff.

having Paket support in IFSharp is amazing!

**Comment by smoothdeveloper** [comment]
The changes have been released in latest paket 4 alpha release:

* files have moved to .paket/load folder
* the `include.` file name prefix is gone
* when we can assume a default framework, we don't generate a subfolder with framework name (in practice that mean having `framework: single-framework-here` in the `paket.dependencies` file)

**Comment by cgravill** [comment]
I've got this form working on #140 so it's possible to now do:

```
#load "Paket.fsx"
Paket.Package
  [ "MathNet.Numerics"
    "MathNet.Numerics.FSharp"
  ]
#load @".paket/load/main.group.fsx"
```

Paket 4.x brings some API changes which might affect some users, see the pull request for the breaking change.

**Comment by cgravill** [comment]
I have merged #140 which addresses this.
